plugins/catalog-import/src/api/CatalogImportClient.ts

```diff
@@ -124,11 +132,13 @@ export class CatalogImportClient implements CatalogImportApi {
   }: {
     repo: string;
   }): Promise<PartialEntity[]> {
+    const idToken = await this.identityApi.getIdToken();
     const response = await fetch(
       `${await this.discoveryApi.getBaseUrl('catalog')}/analyze-location`,
       {
         headers: {
           'Content-Type': 'application/json',
+          ...(idToken && { Authorization: `Bearer ${idToken}` }),
```

Comment:
we are being inconsistent within the pr whether we use upper- or lowercase A in Authorization, but of course it really doesn't matter